EpikCord/__init__.py

```diff
@@ -77,9 +77,7 @@
 
 """
 :license:
-Some parts of the code is sourced from discord.py
```

Comment:
Bruh

Reply:
why

Reply:
Some parts *are* sourced from DPY, removing this is a rude thing to do to a doctor who cares so much about the community to balance maintaining one of the biggest Discord API Wrappers along with a job.